metashare/api/views.py

```diff
@@ -17,3 +18,8 @@ def get_queryset(self):
 
     def get_object(self):
         return self.get_queryset().get()
+
+
+class ProductViewSet(viewsets.ModelViewSet):
+    queryset = Product.objects.all()
+    serializer_class = ProductSerializer
```

Comment:
@wlonk This endpoint should only return products that are in the DB _and that the logged-in user has contributor access to on GitHub_. I don't see anything related to the latter requirement in this PR?

Reply:
There's a distinction between access to _create new products_ and access to view and contribute to products, I think? As far as I'm aware the details haven't been nailed down yet, but contributor access seems like the place to start for this story at least.